scala-package/README.md

```diff
@@ -1,67 +1,119 @@
   in Scala, Java and other languages built on JVM.
 - It also enables you to construct and customize the state-of-art deep learning models in JVM languages,
   and apply them to tasks such as image classification and data science challenges.
+- The Scala/Java Inferece APIs provides an easy out of the box solution for loading pre-trained MXNet models and running inference on them.
   
 Install
 ------------
- 
-Technically, all you need is the `mxnet-full_2.11-{arch}-{xpu}-{version}.jar` in your classpath.
-It will automatically extract the native library to a tempfile and load it.
-You can find the pre-built jar file in [here](https://search.maven.org/search?q=g:org.apache.mxnet)
- and also our nightly build package [here](https://repository.apache.org/#nexus-search;gav~org.apache.mxnet~)
 
-Currently we provide `linux-x86_64-gpu`, `linux-x86_64-cpu` and `osx-x86_64-cpu`. Support for Windows will come soon.
-Use the following dependency in maven, change the artifactId according to your own architecture, e.g., `mxnet-full_2.11-osx-x86_64-cpu` for OSX (and cpu-only).
+The MXNet Scala/Java packages can be easily included in your Maven managed project.
+The stable jar files for the packages are available on the [MXNet Maven Package Repository](https://search.maven.org/search?q=g:org.apache.mxnet)
+Currently we provide packages for Linux (CPU and GPU) and macOS (CPU only). Support for Windows will come soon.
+To add MXNet Scala/Java package to your project, add the dependency as shown below corresponding to your platform, under the ```dependencies``` tag in your project's ```pom.xml``` :
 
+**Linux GPU**
 ```HTML
 <dependency>
   <groupId>org.apache.mxnet</groupId>
-  <artifactId>mxnet-full_2.10-linux-x86_64-gpu</artifactId>
-  <version>0.1.1</version>
+  <artifactId>mxnet-full_2.11-linux-x86_64-gpu</artifactId>
+  <version><latest-mxnet-version></version>
 </dependency>
 ```
 
-You can also use `mxnet-core_2.10-0.1.1.jar` and put the compiled native library somewhere in your load path.
+**Linux CPU**
 
 ```HTML
 <dependency>
   <groupId>org.apache.mxnet</groupId>
-  <artifactId>mxnet-core_2.10</artifactId>
-  <version>0.1.1</version>
+  <artifactId>mxnet-full_2.11-linux-x86_64-cpu</artifactId>
+  <version><latest-mxnet-version></version>
 </dependency>
 ```
 
-If you have some native libraries conflict with the ones in the provided 'full' jar (e.g., you use openblas instead of atlas), this is a recommended way.
-Refer to the next section for how to build it from the very source.
+**macOS CPU**
+```HTML
+<dependency>
+  <groupId>org.apache.mxnet</groupId>
+  <artifactId>mxnet-full_2.11-osx-x86_64-cpu</artifactId>
+  <version><latest-mxnet-version></version>
+</dependency>
+```
+
+The ```<latest-mxnet-version>``` is a placeholder corresponding to the current MXNet version. The latest version information will be available on the [MXNet Maven Package Repository](https://search.maven.org/search?q=g:org.apache.mxnet). 
```

Comment:
Instead of using a placeholder, I think we should use something like "[1.4.0,)". This automatically picks up the latest version. We can also do "[1.4.0-SNAPSHOT,)" for nightly.

Reply:
Is this only for documentation purpose or does it work in the pom file?